src/Microsoft.Framework.PackageManager/Bundle/BundleManager.cs

```diff
@@ -284,5 +289,49 @@ private DependencyContext CreateDependencyContext(Runtime.Project project, Frame
             dependencyContext.Walk(project.Name, project.Version);
             return dependencyContext;
         }
+
+        private bool ResolveActualRuntimeNames(IList<string> runtimes)
+        {
+            for (int i = 0; i < runtimes.Count(); i++)
+            {
+                if (string.Equals("active", runtimes[i], StringComparison.OrdinalIgnoreCase))
+                {
+                    string activeRuntime = GetActiveRuntimeName();
```

Comment:
The `GetActiveRuntime()` could be called multiple times. Can its result be cached?